packages/auth/e2e/auth.e2e.js

```diff
@@ -58,6 +58,47 @@ describe('auth()', () => {
     });
   });
 
+  describe('reload()', () => {
+    it('Meta data returns as expected with annonymous sign in', async () => {
+      await firebase.auth().signInAnonymously();
+      await Utils.sleep(50);
+      const firstUser = firebase.auth().currentUser;
+      await firstUser.reload();
+
+      await firebase.auth().signOut();
+
+      await firebase.auth().signInAnonymously();
+      await Utils.sleep(50);
+      const secondUser = firebase.auth().currentUser;
+      await secondUser.reload();
+
+      firstUser.metadata.creationTime.should.not.equal(secondUser.metadata.creationTime);
+    });
+
+    it('Meta data returns as expected with email and password sign in', async () => {
+      const random = Utils.randString(12, '#aA');
+      const email1 = `${random}@${random}.com`;
+      const pass = random;
+
+      await firebase.auth().createUserWithEmailAndPassword(email1, pass);
+      await Utils.sleep(100);
```

Comment:
What's this sleep needed for?

Reply:
Current user was unavailable, it appeared that the test was too quick. Seemed a bit of an anomaly at the time.

Happy to retest...?

Reply:
PR updated